eclipse-pass/pass-docker
# Update images to use 0.1.0 codebase

Step 1 for https://github.com/eclipse-pass/main/issues/290

* Bump indexer code to 0.1.0 and esconfig 3.5
  * Indexer now pulls compiled JAR from Maven Central, instead of Github
  * `docker-compose: indexer/image` ref should be updated to include the image hash, when available
* Update `fcrepo` dependencies to 0.1.0 and moved to `org.eclipse.pass` groupId
  * Updated: `jsonld-addon-filters`, `pass-authz-core`, `pass-authz-roles`, `pass-user-service`, `jms-addons`
* Update `authz` image
* Update `doi` image
* Rebuild and update `activemq`, `ftpserver`, `proxy`, `idp`, `ldap`, `sp`, `dspace` images
* Rebuild and update `postgres`, since it has some custom configs
* Update the Go services: `download-service`, `policy-service`, `metadata-schemas`
* Update Deposit Services
* Update notification services

## Testing

Run our ITs from our various projects?

* Change the image tags referenced in various projects
* Run their ITs against the new images

ITs run against various projects listed here: https://github.com/eclipse-pass/main/issues/268#issuecomment-1190447487

---

We should also do some sanity checks in this repo by spinning up the full stack and poking things for a bit

``` sh
docker-compose pull
docker-compose up -d
# Optional to see how spammy the logs can be
docker-compose logs -f
```
